simulation_parameters/microbe_stage/biomes.json

```diff
@@ -2887,8 +3048,8 @@
           "Ambient": 0
         },
         "hydrogensulfide": {
-          "Amount": 0,
-          "Density": 0,
+          "Amount": 12500,
```

Comment:
Why was hydrogensulfide added to the seafloor patch?

Reply:
Then it would make more sense for sulfur to spread from the vents patch instead of being harcoded like that. But there's not enough time before tomorrow to include this change for 0.8.0.